CMakeLists.txt

```diff
@@ -98,9 +98,10 @@ if (${TEST})
     set(TEST_NAME ${PRODUCT_NAME}_test)
 
     set(TEST_FILES
-        test/common/util.c
+        src/cpu.c
         test/fixtures/cpu_mmu.c
-        test/alu_8bit_sub_test.c)
+        test/alu_8bit_sub_test.c
+        test/alu_8bit_sbc_test.c)
 
     set(TEST_SOURCES ${TEST_FILES} ${SRC_FILES})
```

Comment:
There's no need to include `src/cpu.c` in `TEST_FILES` since `TEST_SOURCES` includes `cpu.c`